src/diffusers/models/attention.py

```diff
@@ -253,6 +253,8 @@ def __init__(
             self.norm3 = nn.LayerNorm(dim, norm_eps, norm_elementwise_affine)
         elif norm_type == "layer_norm_i2vgen":
             self.norm3 = None
+        elif norm_type == "layer_norm_latte":
```

Comment:
ohh I guess here we just follow suit with  `layer_norm_i2vgen` but `norm_type` should indicate the type of normalization layer, i.e. "ada_norm_zero", "ada_norm", "layer_norm", "ada_norm_continuous"; if we just want to configure whether to have norm3 or not, we should use a new argument for that 

cc @DN6 here